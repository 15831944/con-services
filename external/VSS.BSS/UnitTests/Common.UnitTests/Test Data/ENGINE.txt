<?xml version="1.0" encoding="UTF-8"?>
<engine:topic xmlns:engine="http://www.cat.com/em/feed/v4/EngineParameters" xmlns:feed="http://www.cat.com/em/feed/v4/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v4/EngineParameters https://emfeed.rd.cat.com/EMFeedFiles/schema/v4/EngineParameters.xsd">
   
<nextBuffer>
   <url>https://emfeed.rd.cat.com/EMFeedService/v4/feed/nighthawk/ENGINE/12252790</url>
   <moreData>false</moreData>
</nextBuffer>
<message>
      <header>
         <messageId>12250362</messageId>

         <masterMsgId>5518741</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DATA_FEED_TEST08783</moduleCode>
         <moduleTime>2009-02-18 15:51:54</moduleTime>
         <receivedTime>2009-02-18 15:51:54</receivedTime>
      </header>
      <maxFuel>
         <value>76191</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>3678</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>218103969</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1076</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>16249914</value>
         <uom>Count</uom>
      </revolutions>
   </message>
<message>
      <header>
         <messageId>12250388</messageId>

         <masterMsgId>5518763</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DATA_FEED_TEST08783</moduleCode>
         <moduleTime>2009-02-18 15:51:58</moduleTime>
         <receivedTime>2009-02-18 15:51:58</receivedTime>
      </header>
      <maxFuel>
         <value>0</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>312</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>27572</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1345</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>34826758</value>
         <uom>Count</uom>
      </revolutions>
   </message>
   <message>
      <header>
         <messageId>12250388</messageId>

         <masterMsgId>5518763</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DQ0200020816Q1</moduleCode>
         <moduleTime>2009-02-18 15:51:58</moduleTime>
         <receivedTime>2009-02-18 15:51:58</receivedTime>
      </header>
      <maxFuel>
         <value>0</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>312</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>27572</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1345</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>34826758</value>
         <uom>Count</uom>
      </revolutions>
   </message>
   <message>
      <header>
         <messageId>12250388</messageId>

         <masterMsgId>5518763</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-02-18 15:51:58</moduleTime>
         <receivedTime>2009-02-18 15:51:58</receivedTime>
      </header>
      <maxFuel>
         <value>0</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>312</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>27572</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1345</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>34826758</value>
         <uom>Count</uom>
      </revolutions>
   </message>
   <message>
      <header>
         <messageId>12250388</messageId>

         <masterMsgId>5518763</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2009-02-18 15:51:58</moduleTime>
         <receivedTime>2009-02-18 15:51:58</receivedTime>
      </header>
      <maxFuel>
         <value>0</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>312</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>27572</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1345</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>34826758</value>
         <uom>Count</uom>
      </revolutions>
   </message>
   <message>
      <header>
         <messageId>12250388</messageId>

         <masterMsgId>5518763</masterMsgId>
         <equipment>
            <serialNumber>-1CF08783</serialNumber>
            <make>CAT</make>
            <model>DFTEST_723</model>
            <nickname>NICK_723</nickname>

         </equipment>
         <moduleCode>-DQ0200079955DQ1</moduleCode>
         <moduleTime>2019-02-18 15:51:58</moduleTime>
         <receivedTime>2009-02-18 15:51:58</receivedTime>
      </header>
      <maxFuel>
         <value>0</value>

         <uom>Gallons</uom>
      </maxFuel>
      <idleFuel>
         <value>312</value>
         <uom>Gallons</uom>
      </idleFuel>
      <idleTime>

         <value>27572</value>
         <uom>Hours</uom>
      </idleTime>
      <starts>
         <value>1345</value>
         <uom>Count</uom>
      </starts>

      <revolutions>
         <value>34826758</value>
         <uom>Count</uom>
      </revolutions>
   </message>
</engine:topic>

