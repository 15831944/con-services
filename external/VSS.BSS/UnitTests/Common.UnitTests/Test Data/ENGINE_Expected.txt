<?xml version="1.0" encoding="utf-8"?>
<ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance"
xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10">
  <NHDataWrapper>
    <Data z:Id="i1"
    xmlns:d3p1="http://nighthawk.vss.com/data/2009/03"
    i:type="d3p1:DataEngineParameters"
    xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/">
      <EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data"
      i:nil="true"
      xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" />
      <d3p1:AssetID>0</d3p1:AssetID>
      <d3p1:ConsumptionGallons i:nil="true" />
      <d3p1:DebugRefID>12250362</d3p1:DebugRefID>
      <d3p1:DeviceType>ProductLink</d3p1:DeviceType>
      <d3p1:DimSource i:nil="true" />
      <d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses">

        <d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data"
        i:nil="true" />
      </d3p1:DimSourceReference>
      <d3p1:EngineIdleHours>
      10905198.450000001</d3p1:EngineIdleHours>
      <d3p1:EventUTC>2009-02-18T15:51:54</d3p1:EventUTC>
      <d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID>
      <d3p1:ID>0</d3p1:ID>
      <d3p1:IdleFuelGallons>459.75</d3p1:IdleFuelGallons>
      <d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC>
      <d3p1:LevelPercent i:nil="true" />
      <d3p1:MID i:nil="true" />
      <d3p1:MachineIdleFuelGallons i:nil="true" />
      <d3p1:MachineIdleHours i:nil="true" />
      <d3p1:MaxFuelGallons>9523.875</d3p1:MaxFuelGallons>
      <d3p1:Revolutions>16249914</d3p1:Revolutions>
      <d3p1:SourceMsgID>5518741</d3p1:SourceMsgID>
      <d3p1:Starts>1076</d3p1:Starts>
      <d3p1:UpdateUTC>0001-01-01T00:00:00</d3p1:UpdateUTC>
      <d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID>
    </Data>
    <StoreAttempts>0</StoreAttempts>
  </NHDataWrapper>
  <NHDataWrapper>
    <Data z:Id="i2"
    xmlns:d3p1="http://nighthawk.vss.com/data/2009/03"
    i:type="d3p1:DataEngineParameters"
    xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/">
      <EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data"
      i:nil="true"
      xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" />
      <d3p1:AssetID>0</d3p1:AssetID>
      <d3p1:ConsumptionGallons i:nil="true" />
      <d3p1:DebugRefID>12250388</d3p1:DebugRefID>
      <d3p1:DeviceType>ProductLink</d3p1:DeviceType>
      <d3p1:DimSource i:nil="true" />
      <d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses">

        <d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data"
        i:nil="true" />
      </d3p1:DimSourceReference>
      <d3p1:EngineIdleHours>
      1378.6000000000001</d3p1:EngineIdleHours>
      <d3p1:EventUTC>2009-02-18T15:51:58</d3p1:EventUTC>
      <d3p1:GPSDeviceID>-DATA_FEED_TEST08783</d3p1:GPSDeviceID>
      <d3p1:ID>0</d3p1:ID>
      <d3p1:IdleFuelGallons>39</d3p1:IdleFuelGallons>
      <d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC>
      <d3p1:LevelPercent i:nil="true" />
      <d3p1:MID i:nil="true" />
      <d3p1:MachineIdleFuelGallons i:nil="true" />
      <d3p1:MachineIdleHours i:nil="true" />
      <d3p1:MaxFuelGallons i:nil="true" />
      <d3p1:Revolutions>34826758</d3p1:Revolutions>
      <d3p1:SourceMsgID>5518763</d3p1:SourceMsgID>
      <d3p1:Starts>1345</d3p1:Starts>
      <d3p1:UpdateUTC>0001-01-01T00:00:00</d3p1:UpdateUTC>
      <d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID>
    </Data>
    <StoreAttempts>0</StoreAttempts>
  </NHDataWrapper>
  <NHDataWrapper>
    <Data z:Id="i3"
    xmlns:d3p1="http://nighthawk.vss.com/data/2009/03"
    i:type="d3p1:DataEngineParameters"
    xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/">
      <EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data"
      i:nil="true"
      xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" />
      <d3p1:AssetID>0</d3p1:AssetID>
      <d3p1:ConsumptionGallons i:nil="true" />
      <d3p1:DebugRefID>12250388</d3p1:DebugRefID>
      <d3p1:DeviceType>ProductLink</d3p1:DeviceType>
      <d3p1:DimSource i:nil="true" />
      <d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses">

        <d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data"
        i:nil="true" />
      </d3p1:DimSourceReference>
      <d3p1:EngineIdleHours>
      1378.6000000000001</d3p1:EngineIdleHours>
      <d3p1:EventUTC>2009-02-18T15:51:58</d3p1:EventUTC>
      <d3p1:GPSDeviceID>-DQ0200020816Q1</d3p1:GPSDeviceID>
      <d3p1:ID>0</d3p1:ID>
      <d3p1:IdleFuelGallons>39</d3p1:IdleFuelGallons>
      <d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC>
      <d3p1:LevelPercent i:nil="true" />
      <d3p1:MID i:nil="true" />
      <d3p1:MachineIdleFuelGallons i:nil="true" />
      <d3p1:MachineIdleHours i:nil="true" />
      <d3p1:MaxFuelGallons i:nil="true" />
      <d3p1:Revolutions>34826758</d3p1:Revolutions>
      <d3p1:SourceMsgID>5518763</d3p1:SourceMsgID>
      <d3p1:Starts>1345</d3p1:Starts>
      <d3p1:UpdateUTC>0001-01-01T00:00:00</d3p1:UpdateUTC>
      <d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID>
    </Data>
    <StoreAttempts>0</StoreAttempts>
  </NHDataWrapper>
  <NHDataWrapper>
    <Data z:Id="i4"
    xmlns:d3p1="http://nighthawk.vss.com/data/2009/03"
    i:type="d3p1:DataEngineParameters"
    xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/">
      <EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data"
      i:nil="true"
      xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" />
      <d3p1:AssetID>0</d3p1:AssetID>
      <d3p1:ConsumptionGallons i:nil="true" />
      <d3p1:DebugRefID>12250388</d3p1:DebugRefID>
      <d3p1:DeviceType>ProductLink</d3p1:DeviceType>
      <d3p1:DimSource i:nil="true" />
      <d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses">

        <d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data"
        i:nil="true" />
      </d3p1:DimSourceReference>
      <d3p1:EngineIdleHours>
      1378.6000000000001</d3p1:EngineIdleHours>
      <d3p1:EventUTC>2009-02-18T15:51:58</d3p1:EventUTC>
      <d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID>
      <d3p1:ID>0</d3p1:ID>
      <d3p1:IdleFuelGallons>39</d3p1:IdleFuelGallons>
      <d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC>
      <d3p1:LevelPercent i:nil="true" />
      <d3p1:MID i:nil="true" />
      <d3p1:MachineIdleFuelGallons i:nil="true" />
      <d3p1:MachineIdleHours i:nil="true" />
      <d3p1:MaxFuelGallons i:nil="true" />
      <d3p1:Revolutions>34826758</d3p1:Revolutions>
      <d3p1:SourceMsgID>5518763</d3p1:SourceMsgID>
      <d3p1:Starts>1345</d3p1:Starts>
      <d3p1:UpdateUTC>0001-01-01T00:00:00</d3p1:UpdateUTC>
      <d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID>
    </Data>
    <StoreAttempts>0</StoreAttempts>
  </NHDataWrapper>
  <NHDataWrapper>
    <Data z:Id="i5"
    xmlns:d3p1="http://nighthawk.vss.com/data/2009/03"
    i:type="d3p1:DataEngineParameters"
    xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/">
      <EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data"
      i:nil="true"
      xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" />
      <d3p1:AssetID>0</d3p1:AssetID>
      <d3p1:ConsumptionGallons i:nil="true" />
      <d3p1:DebugRefID>12250388</d3p1:DebugRefID>
      <d3p1:DeviceType>ProductLink</d3p1:DeviceType>
      <d3p1:DimSource i:nil="true" />
      <d3p1:DimSourceReference xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses">

        <d4p1:EntityKey xmlns:d5p1="http://schemas.datacontract.org/2004/07/System.Data"
        i:nil="true" />
      </d3p1:DimSourceReference>
      <d3p1:EngineIdleHours>
      1378.6000000000001</d3p1:EngineIdleHours>
      <d3p1:EventUTC>2009-02-18T15:51:58</d3p1:EventUTC>
      <d3p1:GPSDeviceID>-DQ0200079955DQ1</d3p1:GPSDeviceID>
      <d3p1:ID>0</d3p1:ID>
      <d3p1:IdleFuelGallons>39</d3p1:IdleFuelGallons>
      <d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC>
      <d3p1:LevelPercent i:nil="true" />
      <d3p1:MID i:nil="true" />
      <d3p1:MachineIdleFuelGallons i:nil="true" />
      <d3p1:MachineIdleHours i:nil="true" />
      <d3p1:MaxFuelGallons i:nil="true" />
      <d3p1:Revolutions>34826758</d3p1:Revolutions>
      <d3p1:SourceMsgID>5518763</d3p1:SourceMsgID>
      <d3p1:Starts>1345</d3p1:Starts>
      <d3p1:UpdateUTC>0001-01-01T00:00:00</d3p1:UpdateUTC>
      <d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID>
    </Data>
    <StoreAttempts>0</StoreAttempts>
  </NHDataWrapper>
</ArrayOfNHDataWrapper>
